### ISSUE TITLE
Default Storage

### ISSUE THREAD

**Issue by Itheras** [IceWhaleTech/CasaOS#1517]
Hi i am new to casaos and i find it great but just reading a little i have found something confusing. Many people will be running this from an sbc which will typically have 2 storages one small for the Os and a large one for files. I go tru the install of casaos login and find the drive manager i format the secondary drive then install some docker images immediately find its installing everything on the small drive. Start to search how to default to the big storage drive. This is the information i found : From Casaos,Use the terminal do some changes here and there but beware docker may revert your changes from time to time. User will say go each container setting change it there others will say merge the drive with that followed by a bunch of user saying dont really do that if you value your data because is beta. So all this beautiful Application and ease of use single click interface and I find no way to select a default install drive or drive renaming function "Small observation the drive manager doesn't manage the drives is a formatter at best with no renaming function"so the application  is made with ease of use in mind and click and done but the first important step selection of the storage requires changing things manually in the docker configurations or changes in terminal with a warning that sometimes it can be reverted. Am i missing something ?

Can the drive manager be made to actually manage drives, renaming and default storage selection in the case you dont want to merge drives?.

Thank you. and Great work guys!

**Comment by ETWang1991** [comment]
Thank you for your support and encouragement. Do you mean that you want to decide which hard drive the app files are placed on and which hard drive the user data is placed on? @Itheras

**Comment by ETWang1991** [comment]
If so, we are already thinking about this issue. The situation here is a bit complicated. We will make corresponding functions after sorting it out.

**Comment by Itheras** [comment]
Correct. Example I don't want to use merge I have my sbc and install casaos I later install some storage . if I uninstall the images already installed and reinstall them they still default to the small sbc storage where casa os was originally installed the extra storage is ignored unless I merge the drives or manually change docker configurations on every install .

**Comment by MRDGH2821** [comment]
> If so, we are already thinking about this issue. The situation here is a bit complicated. We will make corresponding functions after sorting it out.

Is there any update?

I found out that `/DATA` can be symlinked to external hard drive, but to me it feels like a risky manevour

**Comment by Duke2030** [comment]
I have the same problem. Started with Casaos,installed home-assistant ,pi-hole and Portainer on it.
Liked it very much but ran out of had-disk space and added a 1TB ssd via sata to it.
Now I don't know how to make that my installed apps can make use of this extra storage.

**Comment by jlgarnier** [comment]
Hi all!

I don't know if this issue was solved but I ran into the same problem:
- CasaOS is installed on a VM with 2 virtual HDDs : sda = 64GB / sdb = 256GB.
- I want applications to be installed along the OS on sda and the DATA folder to point to sdb.
- I couldn't find any setting to configure this mapping.

Thanks in advance for any help!
